ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -211,32 +221,32 @@ private void SceneUW_MouseWheel(object sender, MouseEventArgs e)
             ee.Handled = true;
         }
 
-        // Need to stay here
+        // Need to stay here.
```

Comment:
Why?

Reply:
no idea, old zarby comment that i just added a period to lol